tests/codegen_testcases/solidity/borsh_encoding_complex_types.sol

```diff
@@ -77,7 +77,6 @@ contract EncodingTest {
 	    // CHECK: branch block2
 
         // CHECK: block4: # end_for
-	    // CHECK: ty:bytes %abi_encoded.temp.17 = (alloc bytes len (uint32 8 + %array_bytes_size_0.temp.15))
```

Comment:
This instruction should not be gone. We are not allocating memory for the ecoding anymore. In addition, it should be `uint32 12 + %array_bytes_size_0.temp.15`, because we need 8 bytes for the `uint64` and more four bytes for the `string[]` length.

Reply:
I'll look into this

Reply:
> I'll look into this

Did you find why this instruction is missing?

Reply:
I'm gonna update all codegen tests it again after #1132 is in